WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -30,7 +30,20 @@ import WordPressFlux
         return tableViewController.tableView
     }
 
-    private var syncHelper: WPContentSyncHelper!
+    private var syncHelpers: [ReaderAbstractTopic: WPContentSyncHelper] = [:]
+
+    private var syncHelper: WPContentSyncHelper? {
+        guard let topic = readerTopic else {
+            return nil
+        }
+        let currentHelper = syncHelpers[topic] ?? WPContentSyncHelper()
+
+        if syncHelpers[topic] == nil {
```

Comment:
Not sure if you necessarily need to check for `nil` here since the same instance will be written. Writing should be quicker than doing another lookup in the dictionary anyway.